internal/security/config/command/proxy/deluser/command.go

```diff
@@ -49,6 +50,7 @@ func NewCommand(
 	flagSet.StringVar(&dummy, "confdir", "", "") // handled by bootstrap; duplicated here to prevent arg parsing errors
 
 	flagSet.StringVar(&cmd.username, "user", "", "Username of the user to delete")
+	flagSet.StringVar(&cmd.jwt, "jwt", "", "The token for use when accessing the Kong Admin API")
```

Comment:
See earlier comments about maybe more clearly distinguishing that this is an kong admin jwt file.